bin/test

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+printf "\nRunning 'crystal spec'\n\n"
+crystal spec
+printf "\nRunning 'shards build'\n\n"
+shards build
+printf "\nRunning 'crystal tool format src'\n\n"
```

Comment:
I'm not sure if we want to run ```crystal tool format``` on more than the source directory. I did see some errors when I ran it on the project root.